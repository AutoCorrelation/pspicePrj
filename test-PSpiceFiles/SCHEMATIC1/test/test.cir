** Profile: "SCHEMATIC1-test"  [ c:\users\user\desktop\codespace\pspiceprj\test-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.TRAN  0 4m 0 10u 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
