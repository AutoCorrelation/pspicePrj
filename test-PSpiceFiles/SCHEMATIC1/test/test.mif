 lib=C:\Cadence\SPB_17.2\tools\PSpice\Library\pwrmos.lib, offset=13390, size=334
