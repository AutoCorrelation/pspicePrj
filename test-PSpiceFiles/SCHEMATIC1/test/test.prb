[MEASUREMENTS]
TR 0 Max_XRange(v(vin),0,4m)- Min_XRange(v(vin),0,4m)
TR 0 Max_XRange(v(vout1),0,4m)- Min_XRange(v(vout1),0,4m)
[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
BEGIN TRACE V(VOUT1)
MARKERID 1
TRACEADDEXT
END TRACE V(VOUT1)
BEGIN TRACE V(VIN)
MARKERID 2
TRACEADDEXT
END TRACE V(VIN)
END YAXIS 1
END ANAPLOT 1
BEGIN MEASUREMENTS
SIZE 87
BEGIN MEASUREMENT 1
NAME Max_XRange(v(vin),0,4m)- Min_XRange(v(vin),0,4m)
EVALUATE 1
END MEASUREMENT 1
BEGIN MEASUREMENT 2
NAME Max_XRange(v(vout1),0,4m)- Min_XRange(v(vout1),0,4m)
EVALUATE 1
END MEASUREMENT 2
END MEASUREMENTS
END DISPLAY LAST SESSION
