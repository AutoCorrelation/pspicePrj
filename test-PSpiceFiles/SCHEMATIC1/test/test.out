
**** 06/16/24 22:45:57 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\user\desktop\codespace\pspiceprj\test-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.TRAN  0 4m 0 10u 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source TEST
R_R11         N27590 VOUT1  1.8k TC=0,0 
R_R3         N27730 0  330k TC=0,0 
M_M1         VOUT1 N27730 N28374 N28374 M2N6804
R_R1         VIN N27746  0.5k TC=0,0 
R_R2         N27590 N27730  560k TC=0,0 
V_V2         VIN 0  
+SIN 0 50mV 1k 0 0 0
C_C1         N27746 N27730  100uF  TC=0,0 
V_V3         0 N27590 10V
R_R12         N28374 0  1.8k TC=0,0 
C_C2         0 N28374  10mF  TC=0,0 

**** RESUMING test.cir ****
.END

**** 06/16/24 22:45:57 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\user\desktop\codespace\pspiceprj\test-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               M2N6804         
               PMOS            
       LEVEL    3            
           L    2.000000E-06 
           W    1.2          
         VTO   -3.68         
          KP   10.460000E-06 
       GAMMA    0            
         PHI     .6          
      LAMBDA    0            
          RD     .06652      
          RS     .153        
          RG    4.931        
         RDS  444.400000E+06 
          IS    5.477000E-12 
          JS    0            
           N    3            
          PB     .8          
        PBSW     .8          
         CBD    1.281000E-09 
          CJ    0            
        CJSW    0            
          TT  295.000000E-09 
        CGSO    1.798000E-09 
        CGDO  134.900000E-12 
        CGBO    0            
         TOX  100.000000E-09 
          XJ    0            
          UO  300            
       UCRIT   10.000000E+03 
       DELTA    0            
         ETA    0            
      DIOMOD    1            
         VFB    0            
        LETA    0            
        WETA    0            
          U0    0            
        TEMP    0            
         VDD    5            
       XPART    0            


**** 06/16/24 22:45:57 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\user\desktop\codespace\pspiceprj\test-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  VIN)    0.0000  (VOUT1)   -9.9743  (N27590)  -10.0000 (N27730)   -3.7079     

(N27746)    0.0000 (N28374)    -.0257 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V2         0.000E+00
    V_V3        -2.553E-05

    TOTAL POWER DISSIPATION   2.55E-04  WATTS



          JOB CONCLUDED

**** 06/16/24 22:45:57 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\user\desktop\codespace\pspiceprj\test-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .03
